tests/libnode/node.go

```diff
@@ -137,24 +136,26 @@ const (
 	remove mapAction = "remove"
 )
 
-func patchLabelAnnotationHelper(virtCli kubecli.KubevirtClient, nodeName string, newMap, oldMap map[string]string, mapType mapType) (*k8sv1.Node, error) {
+func patchLabelAnnotationHelper(nodeName string, newMap, oldMap map[string]string, mapType mapType) (*k8sv1.Node, error) {
+	path := "/metadata/" + string(mapType) + "s"
```

Comment:
I think that `fmt.Sprint` is preferred. but not sure it's too important.

Reply:
@nunnatsa Hi, preferred by means of performance or codind style? or both?